doc/usage.rst

```diff
@@ -364,7 +364,7 @@ write a ``(profile ...)`` stanza. For instance:
 
 .. code:: scheme
 
-    (profile dev)
+    (profile performances)
```

Comment:
In that case, I believe that it is an is uncountable noun, and so we should use `performance`. Or maybe we can use a predefined profile name, like `(profile release)`?